### ISSUE TITLE
Documentation Unclear

### ISSUE THREAD

**Issue by qreid0210** [Erdroy/Stationeers.Addons#20]
For someone who is unfamiliar with the workings of Visual Studio and programming languages, I am having considerable difficulty following the documentation to get into building a new add-on. For example, there is a step saying to copy the dll's from the stationeers game folder to Libraries/Stationeers with a read me link. Firstly, the documentation doesnt specify which dll's and from which folders within the stationeers install location as there are many within various subfolders. Second, I have skimmed around various VS folders and the folders that were downloaded after copying the github stuff and have yet to find a Libraries/Staioneers folder. Lastly, the read me link gets redirected to a 404 page, as do other read me links in the documentation. 

Again, maybe its because I am unfamiliar with these things but I feel that the documentation is very lacking in specific directions on what goes where and how to do things.

**Comment by Erdroy** [comment]
Yeah, docs are a bit outdated. I'm doing everything in my spare time (which I usually don't have).

This step is no longer required, currently you just have to set path to the game installation directory in the `Source/Stationeers.VS.props` file.
I've updated that line.
